src/main/java/com/minecolonies/coremod/colony/buildings/AbstractBuildingCrafter.java

```diff
@@ -211,27 +206,6 @@ public View(final IColonyView c, final BlockPos l)
             super(c, l);
         }
 
-        /**
-         * Check if an additional recipe can be added.
-         *
-         * @return true if so.
-         */
-        public boolean canRecipeBeAdded()
-        {
-            return AbstractBuildingCrafter.canBuildingCanLearnMoreRecipes(getBuildingLevel(), super.getRecipes().size());
-        }
-    }
-
-    /**
-     * Check if an additional recipe can be added.
-     *
-     * @param learnedRecipes the learned recipes.
-     * @param buildingLevel  the building level.
-     * @return true if so.
-     */
-    public static boolean canBuildingCanLearnMoreRecipes(final int buildingLevel, final int learnedRecipes)
```

Comment:
There are at least 3 instances in the codebase of this. Can you please remove the other 2?